lib/blueprinter/empty_types.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'helpers/type_helpers'
+
+module Blueprinter
+  EMPTY_COLLECTION = "empty_collection".freeze
+  EMPTY_HASH = "empty_hash".freeze
+  EMPTY_STRING = "empty_string".freeze
+
+  module EmptyTypeHelper
```

Comment:
wondering if this module name should be consistent with the file name `empty_types.rb` or vice versa?